<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculadora</class>
 <widget class="QDialog" name="Calculadora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="display">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>391</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layoutBotones">
    <item row="4" column="2">
     <widget class="QPushButton" name="btn6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="btnsemicolon">
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btn0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>82</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnMas">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnIgual">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
